src/services/loginService.ts

```diff
@@ -32,9 +34,25 @@ export class AzureLoginService {
     }
   }
 
-  public static async interactiveLogin(options: InteractiveLoginOptions): Promise<AuthResponse> {
-    await open("https://microsoft.com/devicelogin");
-    return await interactiveLoginWithAuthResponse(options);
+  public static async interactiveLogin(options?: InteractiveLoginOptions): Promise<AuthResponse> {
+    // await open("https://microsoft.com/devicelogin");
+    // return await interactiveLoginWithAuthResponse(options);
+    var authResp: AuthResponse = {credentials: undefined, subscriptions: []};
+    if(!(options.tokenCache as SimpleFileTokenCache).empty()){
+      console.log("exisitng token");
+      var devOptions = {
+        tokenCache: options.tokenCache as SimpleFileTokenCache
+      }
+      // I don't think DeviceTokenCredentials is what we want... maybe MSITokenCredentials?
```

Comment:
Is this comment still relevant? Do we not want to use `DeviceTokenCredentials`?

Reply:
That is out of date, forgot to remove